client/src/main/java/org/camunda/bpm/client/topic/TopicSubscriptionBuilder.java

```diff
@@ -96,6 +98,20 @@
    */
   TopicSubscriptionBuilder processDefinitionVersionTag(String processDefinitionVersionTag);
 
+  /**
+   * @param processVariables to search for external tasks that are supposed to be
+   *                         fetched and locked
```

Comment:
Can we rephrase the Javadocs to point out that using the filter leads to only fetch and lock external tasks in which the specified variables are present?

Reply:
Sure @tasso94 
Kindly suggest if you think below phrase fits well: "processVariable to use as a filter and fetch and lock external tasks with matching variables present in their process instances"

Reply:
How about "processVariables of which the external tasks to be retrieved are related to"

Reply:
Sure and Thanks @tasso94